adflow/pyADflow.py

```diff
@@ -2546,10 +2542,12 @@ def setAeroProblem(self, aeroProblem, releaseAdjointMemory=True):
         # Tell the user if we are switching aeroProblems
         if self.curAP != aeroProblem:
             newAP = True
-            if self.comm.rank == 0:
```

Comment:
why did you remove the comm.rank check here? I am guessing the current version will print this on all procs in comm now?

Reply:
With the `baseclasses` refactoring, a [`pp` function that use pretty print on the sole root proc has been added](https://github.com/mdolab/baseclasses/blob/7ed195c5064b489325f74e4c6b551123b3b67a61/baseclasses/BaseSolver.py#L227-L238).
It is true however that the [pprint import](https://github.com/mdolab/adflow/blob/a310c0f8484b88dd74162de44f8f298d9be87b5b/adflow/pyADflow.py#L35) in `pyADflow.py` is ambiguous at this point, and we might want to remove it altogether and use the provided `baseclasses` method.
@anilyil @nwu63 @sseraj does it make sense?

Reply:
oh thats cool. yeah the import in pyadflow is misleading